Lib/extractor/formats/opentype.py

```diff
@@ -1085,3 +1089,113 @@ def extractOpenTypeFeatures(source):
     if _haveFontFeatures:
         return unparse(source).asFea()
     return ""
+
+
+# -------
+# Anchors
+# -------
+
+
+def extractAnchors(source, destination):
+    if "GPOS" not in source:
+        return
+
+    gpos = source["GPOS"].table
+    # get an ordered list of scripts
+    scriptOrder = _makeScriptOrder(gpos)
+    # extract anchors from each applicable lookup
+    anchorGroups = _gatherAnchorDataFromLookups(gpos, scriptOrder)
+    #print(anchorGroups)
+
+    for groupIndex, groupAnchors in enumerate(anchorGroups):
+        baseAnchors = groupAnchors["baseAnchors"]
+        markAnchors = groupAnchors["markAnchors"]
+
+        for base in baseAnchors.keys():
+            try:
+                anchor = Anchor(destination[base], {"x": baseAnchors[base]["x"], "y": baseAnchors[base]["y"], "name": f"Anchor-{groupIndex}"})
+                destination[base].appendAnchor(anchor)
+            except:
+                continue
+        for mark in markAnchors.keys():
+            try:
+                anchor = Anchor(destination[mark], {"x": markAnchors[mark]["x"], "y": markAnchors[mark]["y"], "name": f"_Anchor-{groupIndex}"})
+                destination[mark].appendAnchor(anchor)
+            except:
+                continue
```

Comment:
You can rewrite this without using the defcon.Anchor object and thus get rid of the defcon dependency:

```py
        for base in baseAnchors.keys():
            try:
                anchor = {
                    "x": baseAnchors[base]["x"],
                    "y": baseAnchors[base]["y"],
                    "name": f"Anchor-{groupIndex}",
                }
                destination[base].appendAnchor(anchor)
            except:
                continue
        for mark in markAnchors.keys():
            try:
                anchor = {
                    "x": markAnchors[mark]["x"],
                    "y": markAnchors[mark]["y"],
                    "name": f"_Anchor-{groupIndex}",
                }
                destination[mark].appendAnchor(anchor)
            except:
                continue
```

Reply:
I wonder about the `try...except` statements here. In what case could it go wrong, and would we want to know about it instead of silently ignoring the anchors?

Reply:
I guess you're right. I didn't want my additions to potentially break the code, but I suppose it's better to fix it properly if something breaks instead of pretending like nothing happened.

Reply:
> You can rewrite this without using the defcon.Anchor object and thus get rid of the defcon dependency:

Done. Also rebased on the current master and force-pushed everything in a single commit.